src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/VariantsSparkSink.java

```diff
@@ -118,7 +118,7 @@ private static void writeVariantsSingle(
         }
 
         final JavaRDD<VariantContext> sortedVariants = sortVariants(variants, header, numReducers);
-        final String outputPartsDirectory = outputFile + ".parts";
```

Comment:
Would you mind adding a test that writes variants to gcs so we don't break this again?  There's a `PrintReadsSparkIntegrationTest` that does it for bams, a `ReadSparkSinkUnit` test would be ideal.